components/sensor/pzem004t.rst

```diff
@@ -54,11 +56,25 @@ Configuration variables:
   :ref:`Sensor <config-sensor>`.
 - **voltage** (*Optional*): Use the voltage value of the sensor in volts.
   All options from :ref:`Sensor <config-sensor>`.
+- **energy** (*Optional*): Use the accumulated energy value of the sensor in watt per hour.
+  All options from :ref:`Sensor <config-sensor>`.
 - **update_interval** (*Optional*, :ref:`config-time`): The interval to check the
   sensor. Defaults to ``60s``.
 - **uart_id** (*Optional*, :ref:`config-id`): Manually specify the ID of the :ref:`UART Component <uart>` if you want
   to use multiple UART buses.
 
+Example config for energy
+-------------------------
+.. code-block:: yaml
```

Comment:
This needs more explanation.

How should a user know this is a Home Assistant YAML entry.

A bit of text explaining that the energy value is the total accumulated over lifetime, and this is how to integrate it well into HA would be good.